update-extism.sh

```diff
@@ -0,0 +1,33 @@
+EXTISM_VERSION=$(curl https://api.github.com/repos/extism/extism/releases/latest | jq -r '.name')
```

Comment:
May want to add a shebang and `set -e` on the top of this script so it fails if something goes wrong.